tools/bridge/README-dev.md

```diff
@@ -57,3 +57,29 @@ and open a PR for that branch.
 
 After a release has been made, we should upgrade the version on the
 master branch with `bumpversion patch`.
+
+## Release Checklist
+
+0. Prerequsisite: The master branch contains the version to be
+   released
+1. Open and merge a PR master -> pre-release (no review needed). This
+   will built the docker image and push it to Docker Hub under
+   `trustlines/bridge-next:pre-release`.
+2. Test the pre-release:
+   - Pull the newly built image from Docker Hub.
+   - Check that the bridge config file looks as expected.
```

Comment:
What bridge-config file?

Reply:
Ah, your right, I was thinking about these https://github.com/trustlines-protocol/blockchain/blob/master/tools/quickstart/quickstart/configs/laika/bridge-config.toml

But apparently we don't distribute them as part of the bridge, just in the quickstart script. I'll remove this, as well as the last sentence about testnets (which doesn't make sense if we don't have anything testnet related in there).